### DISCUSSION TITLE
What's the root password from the elementary os 6.0 live image build?

### DISCUSSION THREAD

**Discussion by aggalex** [elementary/os#427] [Q&A] [answered]
I successfully built elementary OS 6.0, but the user has a root password, which makes me unable to connect to Wifi, test packages, etc. What's the password?

**Comment by dgk4201** [comment]
I have the same issue. I downloaded the sd img, booted and get a "elementary login:" followed by a password request.
How do I boot/install the system?

**Comment by ForumPlayer** [comment]
No answers yet?
I'm trying to find by myself, if i find before someone answering here, i'll let y'all know

**Comment by jeremypw** [comment]
As elementary is based on Ubuntu I imagine there is no default root password.  Normally you would boot into the live image and get an option to run it or install it.  During installation you set up a admin user that can perform root operations via sudo. I admit I have only ever installed it.

**Comment by jeremypw** [comment, marked as answer]
Just tried booting into an elementary image on USB (using machine boot options and choosing "Try or install" option). After briefly showing a tty with "elementary login:", a graphical wizard to choose language and keyboard settings was shown and then an option to to run "Demo Mode".  This started a live session without needing a password.  The Switchboard showed the user was "Live Session User" with administrative rights.  It was possible to change the password of this user without needing to enter a password, but none of the other operations I tried (like running `sudo` or connecting to to a wireless network) needed a password anyway.  So you should not need the root password in order to run a live session or install elementary as long as you run the graphical installer and choose "Demo Mode" or one of the install options.

**Reply by ForumPlayer** [reply]
My problem was:
I have a friend who is starting on her Linux journey, so I suggested the easiest distro I know (also the one I started on).
We had some problems on her VM and the screen wasn't picking an "usable" resolution, so it was impossible to do anything (even click the "Try now").
Since I have some experience with Linux (just 6y using), I through of trying to fix the resolution directly in the `/etc/X11/`, but to this, I had to do it through the TTY.
That's how I came here.

By now, I found a solution for her problem (it really was on the VM), but having access to the TTY would also help in a case of this happening in a real machine (not a VM).

Due to the inability of even starting the "Demo mode" to have access to the terminal, we couldn't just do a `sudo passwd` and then changing to the TTY.
Also if I could get access to the terminal, even with an unusable resolution, I could just run a SSH server and troubleshoot it remotely and then install it after the X11 problem being solved. (I hope it all make any sense for you, since I'm the worst one to explain things)

**Comment by shqear93** [comment]
same issue here! any resolution? I was about to pay for it and it's not even installable?

**Reply by ForumPlayer** [reply]
Are you trying to install in a VM or a normal PC/Server?

**Reply by shqear93** [reply]
@ForumPlayer I was trying to install on my new laptop HUAWEI MateBook X Pro

**Comment by balweshpuram** [comment]
Just wait few minutes, the login screen automatically redirect to the next step..!

**Comment by Nandan-N** [comment]
Had the same problem. Just had to wait a while for it to redirect.

**Comment by iFloris** [comment]
This is happening my machine too, a HP ProBook 450 15.6" G9 using Horus Elementary on first boot. 
However, the machine does not seem to move beyond the login prompt. I have left it for a little over an hour and I am assuming this is not supposed to take that long. I have verified the MD5 and used Etcher to write the iso as per the installation instructions. What might I try to get the installation going?

Update: I was able to get to the installer using the safe graphics option. However, after installation I am again met by the login prompt.

**Comment by librafrog** [comment]
I think y'all should be fine, either wait the input screen, or leave field with no password/blank. 💁🏼‍♂️

**Comment by destroythexlight** [comment]
For anyone landing here having the issue where it's getting stuck on the login screen, you're most likely seeing some incompatibility with your graphic driver. That said, you can't try to fix it without being able to log in :)
The username is either `ubuntu` or `elementary`. There should be no password.

**Comment by tismofied** [comment]
> For anyone landing here having the issue where it's getting stuck on the login screen, you're most likely seeing some incompatibility with your graphic driver. That said, you can't try to fix it without being able to log in :)
> 
> The username is either `ubuntu` or `elementary`. There should be no password.

None of these worked for me. 😞

**Comment by pferrucci** [comment]
On ElementaryOS live 8.0, login is "elementary", no password.

**Reply by BexaGal** [reply]
Thanks! Log on! Although, for how long should I wait until installer loads itself in non-save graphics mode in Hyper-V (2gen)?
